src/main/scala/io/iohk/ethereum/utils/Config.scala

```diff
@@ -38,7 +38,7 @@ object Config {
   object Network {
     private val networkConfig = config.getConfig("network")
 
-    val protocolVersion = networkConfig.getString("protocol-version")
+    val protocolVersion = networkConfig.getInt("protocol-version")
```

Comment:
Maybe I misunderstood something, but it seems the network id is an `Int` in the [config file](https://github.com/input-output-hk/etc-client/pull/279/files#diff-6bb9146203d6ddeb4f8d799d28054c96R62).
Maybe it would be better to have them both as a `String` instead? AFAIK there's no strong reason for having them being `Int` or `String`, so they could be changed either way.